src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -177,9 +178,11 @@ class ReimbursementAccountPage extends React.Component {
                 CONST.BANK_ACCOUNT.STEP.ACH_CONTRACT,
             ], currentStep);
             return (
-                <ReimbursementAccountLoadingIndicator
-                    isSubmittingVerificationsData={isSubmittingVerificationsData}
-                />
+                <FullPageOfflineBlockingView>
```

Comment:
Do we mind that the header is no longer shown to the user? They can exit the modal by clicking outside of it but I don't think that's consistent with other screens that show the full page offline view.

![Screen Shot 2022-09-29 at 2 03 13 PM](https://user-images.githubusercontent.com/12268372/193003566-084c7689-339d-4e75-b89a-fa0cdf99f35a.png)

How about updating ReimbursementAccountLoadingIndicator to wrap [this](https://github.com/Expensify/App/blob/0162853b4f36240abf1d96924495556f1ebdb18d/src/components/ReimbursementAccountLoadingIndicator.js#L26-L42) in the FullPageOfflineBlockingView? 

![Screen Shot 2022-09-29 at 2 08 06 PM](https://user-images.githubusercontent.com/12268372/193004141-c207523d-8337-47ed-8080-490ebc6dd37c.png)

Reply:
>Do we mind that the header is no longer shown to the user? They can exit the modal by clicking outside of it but I don't think that's consistent with other screens that show the full page offline view.

Totally agree, it should have a header.